src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -145,6 +148,7 @@ const incorrectDateFormatMessage =
   "Only one of these date formats is allowed: " +
   "['dd', 'mm', 'yyyy'], " +
   "['mm', 'dd', 'yyyy'], " +
+  "['yyyy', 'mm', 'dd'], " +
```

Comment:
**suggestion(non-blocking):** You'll need to update the comment for `dateFormat` so that the prop tables have the correct description. Lines 82 -> 89.